src/5.0_to_5.1/createFilterWithFirstMatchingCube.ts

```diff
@@ -0,0 +1,30 @@
+import { stringify } from "@activeviam/activeui-sdk-5.0";
+import {
+  createFilter,
+  MdxExpression,
+  Filter,
+  Cube,
+} from "@activeviam/activeui-sdk-5.1";
+
+/**
+ * Returns a {@link Filter} based on the input `mdx`, with the first matching cube.
```

Comment:
```suggestion
 * Returns a {@link Filter} based on `mdx`, with the first matching cube.
```
Also, can we be a bit more specific on what "matching" means here?